### PR TITLE
Add new line in each log

### PR DESCRIPTION
**Pull request by nprizal**
Some logs doesn't have a new line. This PR adds new line to all logs.

### CODE REVIEW

**Comment by wooly** [comment]
I _think_ we should change the base of this branch to 0.5.x

**Review by wooly — APPROVED** [review]

**Review by amybiyuliu — APPROVED** [review]
Thanks for adding this!

File: main.go [hunk 1]
```diff
@@ -170,6 +170,7 @@ func retryFailedTests(testRunner runner.Rspec, maxRetries int) int {
 // logErrorAndExit logs an error message and exits with the given exit code.
 func logErrorAndExit(exitCode int, format string, v ...any) {
 	fmt.Printf(format, v...)
+	fmt.Print("\n")
```
Comment (DrJosh9000) [inline main.go:173]: Drive-by suggestion:

1. A shorter way to write this is `fmt.Println()`
2. Alternatively you could use `format+"\n"` in the line above

Comment (wooly) [reply main.go:173]: We love driveby suggestions, thanks @DrJosh9000! @nprizal want to make the change and ship?
